docs/data/joy/getting-started/templates/TemplateCollection.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import LZString from 'lz-string';
+import startCase from 'lodash/startCase';
+import NextLink from 'next/link';
```

Comment:
Shouldn't we import from?

```suggestion
import Link from 'docs/src/modules/components/Link';
```

https://mui.com/material-ui/guides/routing/#next-js